ui/src/components/HealthItem.js

```diff
@@ -132,15 +143,21 @@ const HealthItem = ({
             </NonHealthyPopUp>
           }
         >
-          <Link to="/alerts" data-testid="alert-link">
+          <div
+            onClick={() => {
+              history.push('/alerts');
```

Comment:
could be removed, the openLink() already take care of this. 
```suggestion

```

Reply:
Please use `history.replace` instead of `history.push`

And clean up the commits with the best practice of the commit message. 

Then we are good to go.

Thanks!

Reply:
To be precise the `history.replace` should be called after `openLink`.
This is necessary because openLink triggers a change on `shell-ui` history which renders the same federated component coming from Metalk8s (/ is matched for /alerts) then Metalk8s federated UI doesn't detect any changes in it's history and doesn't rerender.
Using `history.replace` before calling openLink would replace the current browsed page history. Calling it after would replace the change performed by openLink, so you should be browsed to the Alert page and pressing once the back button should bring you back to the overview page.